server/src/main/java/io/crate/execution/dsl/projection/builder/InputColumns.java

```diff
@@ -58,7 +58,7 @@
 /**
  * Provides functions to create {@link InputColumn}s
  */
-public final class InputColumns extends DefaultTraversalSymbolVisitor<InputColumns.SourceSymbols, Symbol> {
+public final class InputColumns extends SymbolVisitor<InputColumns.SourceSymbols, Symbol> {
```

Comment:
`InputColumns` doesn't rely on the default traversal mechanism as far as I can tell